tests/end-to-end/backup/pkg/tests/eventmesh/eventmesh.go

```diff
@@ -219,8 +225,12 @@ func (f *eventMeshFlow) WaitForBroker() error {
 	return WaitForBroker(f.messaging, f.brokerName, f.namespace, retry.Delay(10*time.Second), retry.DelayType(retry.FixedDelay), retry.Attempts(10))
 }
 
-func (f *eventMeshFlow) PublishTestEvent() error {
-	return SendEvent(fmt.Sprintf("http://%s.%s.svc.cluster.local", f.applicationName, integrationNamespace), "Dumbidu", f.eventType, f.eventTypeVersion)
+func (f *eventMeshFlow) PublishTestEventToMesh() error {
```

Comment:
Should we make the event publishing resilient? Like adding a few retries. I see old `PublishTestEvent ` didn't have it as well.  Anyways, nothing we have to do in this PR.

Reply:
might be a good way to improve the test once we refactor it into a standalone test 
see https://github.com/kyma-project/community/pull/442